### DISCUSSION TITLE
Package authorship vs ownership

### DISCUSSION THREAD

**Discussion by philpax** [AmbientRun/Ambient#1023] [Ideas]
The package manifests supports an `authors` field, inspired by `Cargo.toml` and other package manifests. On the website, however, we show the owner of the package, which can be different to the authors.

Using owners allows us to link directly to the profiles, so you can easily see what else they've done. However, I don't think this fully captures the complexity of package authorship: it's not uncommon, especially after years of work, for people to leave the project they started and hand ownership over to someone else, but still keeping authorship credit on the project.

I think we should show both (hiding authors if not present), and allow people to specify the authors that they think are appropriate, and have those also shown.

**Comment by FredrikNoren** [comment]
@philpax So for all UI (i.e. mod manager etc.) I think we should use the current owners (they are the ones that can take action when things are not working etc.). Tbh when it comes to attribution of past collaborators, I kind of think that is easier for people to just put in the README. We could leave the authors field there but I don't think we should really show it anywhere, so then it anyway becomes a bit pointless.

**Reply by philpax** [reply]
Hm. Honestly, it looks like you're right: https://rust-lang.github.io/rfcs/3052-optional-authors-field.html

I'll think about it a bit but maybe we should just remove the field. I'll ask around to see if other communities have any particularly strong feelings about it.
